### ISSUE TITLE
Craft 3 Support

### ISSUE THREAD

**Issue by engram-design** [elivz/VzAddress-Craft#33]
Hi, 

Are you looking to add Craft 3 support soon?

**Comment by elivz** [comment]
Yes, it is in progress. Hoping to have it released around the end of the month.

**Comment by RitterKnightCreative** [comment]
Just curious if you're still planning on doing a V3 version?

**Comment by elivz** [comment]
Yes, sorry for the delay. I've got a beta version of the conversion, and am going to try to get it finished up and released soon. I had a baby right around the time Craft3 came out, and have been playing catch-up ever since...

**Comment by RitterKnightCreative** [comment]
Congrats sir! Family comes first!, totally respect that. Great plugin, BTW.

**Comment by bootsified** [comment]
Congrats, @elivz! Kids certainly make development more challenging. 😉 Your fields have become staples for a lot of us (I've been a fan since my old EE days). I look forward to the update, but hope you find a groove on the homefront soon as well.  Thanks for all you do.  Cheers!  🍻

**Comment by mijar** [comment]
Hi @elivz, congrats! :-) We used this plugin in our Craft 2 project too, thus I am following the thread since we would like to use it in the migrated project too.;) All the best!

**Comment by sjelfull** [comment]
There is a port over at https://github.com/superbigco/craft-vzaddress that you can use while we wait for @elivz to find time.

Mind that I don't have much time to fix anything myself, so feel free to send PRs if you find anything that breaks (the port was done in less than a hour 😬)
